network/transport/v2/handlers.go

```diff
@@ -151,10 +158,114 @@ func (p protocol) handleGossip(peer transport.Peer, msg *Gossip) error {
 	if len(refs) > 0 {
 		// TODO swap for trace logging
 		log.Logger().Infof("received %d new transaction references via Gossip", len(refs))
+		p.sendTransactionListQuery(peer.ID, refs)
 	}
 	p.gManager.GossipReceived(peer.ID, refs...)
 	// TODO compare xor
 	// TODO send new message
 
 	return nil
 }
+
+func (p *protocol) handleTransactionList(peer transport.Peer, envelope *Envelope_TransactionList) error {
+	msg := envelope.TransactionList
+	cid, err := parseConversationID(msg.ConversationID)
```

Comment:
If a brand new node is spawned, is 30 seconds (max. conversation lifetime) enough for the DAG to be sync'd?

Reply:
If it can't do it within this conversation, the protocol should just continue in the next gossip/state sequence with a new conversation.